docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml

```diff
@@ -95,49 +95,49 @@
         </row>
 
         <row>
-          <entry><emphasis>sourcepath</emphasis></entry>
+          <entry><emphasis>sourcePath</emphasis></entry>
 
           <entry>A null-terminated string containing the path and name of the
           file.</entry>
         </row>
 
         <row>
-          <entry><emphasis>maxrecordsize</emphasis></entry>
+          <entry><emphasis>sourceMaxrecordsize</emphasis></entry>
 
           <entry>Optional. An integer containing the maximum size of the
           records in the file. If omitted, the default is 4096.</entry>
         </row>
 
         <row>
-          <entry><emphasis>srcCSVseparate</emphasis></entry>
+          <entry><emphasis>sourceCSVseparate</emphasis></entry>
```

Comment:
In In File.ecl there is a "sourceCsvSeparate", "Csv" instead of "CSV".  Same bellow.